{"name": "test_purchase_order", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\nCase\\test_02_purchase.py:27: AssumptionFailure\n>>\tpytest.assume(num2 - num1 == 1)  # 断言新增之后页面上会多一条数据\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               Case\\test_02_purchase.py:27: AssumptionFailure\nE               >>\tpytest.assume(num2 - num1 == 1)  # 断言新增之后页面上会多一条数据\nE               AssertionError: assert False\n\nD:\\py3\\lib\\site-packages\\six.py:718: FailedAssumption"}, "description": "关联请购单新增采购订单", "start": 1723457628523, "stop": 1723457643917, "uuid": "61cb4123-9bff-4fac-bc07-3611924ea0bc", "historyId": "fe608a92593a4d5c240b1b3bce8adee1", "testCaseId": "d9c45ce0d6cb22409ed0cad5133760d6", "fullName": "Case.test_02_purchase#test_purchase_order", "labels": [{"name": "parentSuite", "value": "Case"}, {"name": "suite", "value": "test_02_purchase"}, {"name": "host", "value": "LAPTOP-J99CE7C3"}, {"name": "thread", "value": "49044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Case.test_02_purchase"}]}